app/components/project-icon.cjsx

```diff
@@ -24,20 +24,23 @@ module.exports = React.createClass
       @getDetails nextProps.project
 
   getDetails: (project) ->
-    apiClient.type 'avatars'
-      .get project.links.avatar.id
-      .then (avatar) =>
-        @setState {avatar}
-      .catch =>
-        project.get 'avatar'
-          .catch =>
-            null
-          .then (avatar) =>
-            @setState {avatar}
+    if project.avatar_src
+      @setState avatar: "http://#{ project.avatar_src }"
+    else
+      apiClient.type 'avatars'
+        .get project.links?.avatar?.id
+        .then (avatar) =>
+          @setState avatar: avatar.src
+        .catch =>
+          project.get 'avatar'
+            .catch =>
+              null
+            .then (avatar) =>
+              @setState avatar: avatar.src
 
```

Comment:
Took out the else block on this one. Now, this should always revert to the default/generic avatar if one isn't found in `avatar_src`. However, I also had to alter the `RecentProjects` component so it would ask for cards on the API call. That seemed to be the only other place where `ProjectIcon` was being used without cards.